tests/cupy_tests/core_tests/test_ndarray_adv_indexing.py

```diff
@@ -326,10 +330,11 @@ def test_invalid_adv_getitem(self, xp):
 @testing.gpu
 class TestArrayInvalidValueAdvGetitemForOldVersion(unittest.TestCase):
 
-    @testing.numpy_cupy_raises()
-    def test_invalid_adv_getitem(self, xp):
-        a = testing.shaped_arange(self.shape, xp)
-        a[self.indexes]
+    def test_invalid_adv_getitem(self):
+        for xp in (numpy, cupy):
+            a = testing.shaped_arange(self.shape, xp)
+            with pytest.raises(IndexError):
+                a[self.indexes]
```

Comment:
The test failed on Jenkins.

Reply:
Removed this test as it is for the old NumPy version.